### DISCUSSION TITLE
Form & Input Content Type Limit Exceed (1024)

### DISCUSSION THREAD

**Discussion by aashish803** [OrchardCMS/OrchardCore#14817] [Q&A]
**### Describe the bug**

ERROR:- " InvalidDataException: Form value count limit 1024 exceeded. "

I am seeing this error in my orchard CMS when I add too many " **forms** " and " **input** " content types on one single page. 
Can you guys show me how to resolve this issue?

Thanks Regards

**Comment by hishamco** [comment]
I think the issue is not related to OC, the HTTP has a limited amount of data to be sent in the same request

**Comment by aashish803** [comment]
> I think the issue is not related to OC, the HTTP has a limited amount of data to be sent in the same request

Is there any way that we can fix this issue, because it is kind of becoming a deal breaker for me.

**Comment by hishamco** [comment]
Try to configure `FormOptions` as the follows:

```csharp
services.Configure<FormOptions>(options =>
{
    options.ValueCountLimit = int.MaxValue;
});
```

**Comment by MikeAlhayek** [comment]
As @hishamco mentioned this isn't an OC specific. This is ASP.NET. You may need to update both `ValueCountLimit` and `ValueLengthLimit`

```
        services.Configure<FormOptions>(options =>
        {
            options.ValueCountLimit = int.MaxValue;
            options.ValueLengthLimit = int.MaxValue;
        });
```
